hw/ip/otbn/dv/model/otbn_model.cc

```diff
@@ -252,10 +252,11 @@ int OtbnModel::start() {
 int OtbnModel::step(svLogic edn_rnd_data_valid,
                     svLogicVecVal *edn_rnd_data, /* logic [255:0] */
                     svLogic edn_urnd_data_valid,
+                    svBitVecVal *status /* bit [7:0] */,
                     svBitVecVal *insn_cnt /* bit [31:0] */,
                     svBitVecVal *err_bits /* bit [31:0] */,
                     svBitVecVal *stop_pc /* bit [31:0] */) {
-  assert(err_bits);
+  assert(edn_rnd_data && err_bits && insn_cnt && err_bits && stop_pc);
```

Comment:
I am a bit confused about this line. Why are we using `edn_rnd_data` here and also wouldn't it fail in between different SW runs where we have `ins_cnt` as 0 (in `otbn_reset` test for example)?

Reply:
This is just checking that the pointers are non-null (which should always be true because of how the DPI-C convention works, but it can't hurt to be careful!)